update

```diff
@@ -1,32 +1,42 @@
 #!/usr/bin/env node
 
-'use strict'
-
-const fs = require('fs')
-const util = require('util')
-const path = require('path')
-const semver = require('semver')
-const ethUtil = require('ethereumjs-util')
-const ipfsImporter = require('ipfs-unixfs-importer')
-const IPLD = require('ipld')
-const inMemory = require('ipld-in-memory')
-const swarmhash = require('swarmhash')
+"use strict";
+
+const fs = require("fs");
+const util = require("util");
```

Comment:
Can you remove any stylistic changes from this file? It's pretty much impossible to review like this.

If you really want to reformat it, you can submit a separate PR but if you do this, please ensure that our `npm run lint` command enforces the same style. Otherwise it won't be maintained.